### PR TITLE
Clean up FOV math, prevent overflow

### PR DESCRIPTION
**Pull request by ceski-1**

### CODE REVIEW

File: src/tables.h [hunk 1]
```diff
@@ -35,6 +35,10 @@
 
 #include "m_fixed.h"
 
+#ifndef M_PI
+#define M_PI 3.14159265358979323846264338327950288
```
Comment (rfomin) [inline src/tables.h:39]: Why do we need this define, `<math.h>` is not included?

Comment (ceski-1) [reply src/tables.h:39]: I always get compilation errors unless I also include SDL.h which seems like too much.

Comment (rfomin) [reply src/tables.h:39]: This is fixed it for me:
```diff
diff --git a/src/r_main.c b/src/r_main.c
index a1a8a4e..3ca7528 100644
--- a/src/r_main.c
+++ b/src/r_main.c
@@ -19,6 +19,9 @@
 //
 //-----------------------------------------------------------------------------
 
+#define _USE_MATH_DEFINES
+#include <math.h>
+
 #include "doomstat.h"
 #include "r_main.h"
 #include "r_things.h"
diff --git a/src/tables.h b/src/tables.h
index b9f9fd5..786cb28 100644
--- a/src/tables.h
+++ b/src/tables.h
@@ -35,10 +35,6 @@
 
 #include "m_fixed.h"
 
-#ifndef M_PI
-#define M_PI 3.14159265358979323846264338327950288
-#endif
-
 #define FINEANGLES              8192
 #define FINEMASK                (FINEANGLES-1)
 
```

Comment (ceski-1) [reply src/tables.h]: That works, thanks. https://github.com/fabiangreffrath/woof/pull/1400/commits/10e845fba51efccee2d6005ec998112d16c8fed3

**Pull request merged**